content/docs/nav.yml

```
@@ -127,27 +127,27 @@
       title: Hook API 참고서
     - id: hooks-faq
       title: Hook 자주 묻는 질문
-- title: Testing
+- title: 테스팅
   items:
     - id: testing
       title: 테스팅 개요
     - id: testing-recipes
-      title: Testing Recipes
+      title: 테스팅 방안
     - id: testing-environments
-      title: Testing Environments
-- title: Concurrent Mode (Experimental)
+      title: 테스팅 환경
+- title: Concurrent 모드(실험적)
```

```suggestion
- title: Concurrent 모드 (실험 단계)
```

공백 추가 및 다른 언어의 번역본을 참고해서 수정 제안드려요.

---

제안하신대로 수정해서 커밋했습니다.